app/models/GoodsRecord.scala

```diff
@@ -26,7 +26,9 @@ final case class GoodsRecord(
   traderReference: String,
   commodityCode: String,
   countryOfOrigin: String,
-  goodsDescription: String
+  goodsDescription: String,
+  category: String,
```

Comment:
So this is a stopgap whilst we work out what we are doing with category - it might be case it. hardcoded as 0 on the router or something, or it's optional, for the moment it's here just to get things working

Reply:
changed to be set in the request automatically